shared/chat/conversation/messages/react-button/index.js

```diff
@@ -204,7 +204,7 @@ export class NewReactionButton extends React.Component<NewReactionButtonProps, N
                 style={iconCastPlatformStyles(
                   Styles.collapseStyles([
                     styles.emojiIconWrapper,
-                    !Styles.isMobile && (this.props.showBorder ? {top: 3} : {top: 1}),
+                    !Styles.isMobile && (this.props.showBorder ? {top: 3} : {top: 0}),
```

Comment:
This looks a little too high next to the message menu button. maybe 1 should stay?